behave/features/steps/other.py

```diff
@@ -28,6 +28,33 @@ def step_impl(context):
     context.uniqueext_used = True
 
 
+@given(u'the {package} package {state} installed on host')
+def step_impl(context, package, state):
+    """
+    Install the package, and uninstall in post- action.  If state is "not", then
+    just check it is not installed.
+    """
+    is_installed, _, _ = run(["rpm", "-q", package])
+    # exit_status 0 => installed
+    is_installed = bool(not is_installed)
+
+    if "not" in state:
+        assert_that(is_installed, equal_to(False))
+        return
+
+    if is_installed:
+        return
+
+    # install the package, and schedule removal
+    def _uninstall_pkg(_context):
+        cmd = ["sudo", "dnf", "-y", "remove", package]
```

Comment:
Hmm, this will work now, but it will stop working when dnf5 replaces dnf - and dnf will be symlink to dnf.
Accepting now, we will cope with that later.